### DISCUSSION TITLE
Project name suggestion...

### DISCUSSION THREAD

**Discussion by emmenlau** [taocpp/taopq#12] [General]
Thanks for your excellent work, the library looks very well done!

Please forgive my humble request, but I find the project name slightly unfortunate. The git repository and library are named `postgres` which is a heavily overloaded name in the realm of PostgreSQL. Since I understand taocpp/postgres is an (elaborate) C++ layer above `libpq`, may I humbly suggest a more distinct name like `taopq` or `libtaopq` or something that identifies it better as a C++ drop-in replacement for `libpq`?

This is really just my two cents, feel free to ignore this humble request.

**Comment by d-frey** [comment]
We used to think about the name only in connection to the project name, but you are probably correct that others will not see it that way, including several tools. I'll consider a change, give me a few days to sleep over it :)

**Comment by emmenlau** [comment]
Thanks for your consideration. Its really nothing major, it just may help to adopt the library for the wide audience... i.e. if its shipped with distributions then a distinct name may be helpful.

**Comment by d-frey** [comment]
It took a bit longer than anticipated, but I finally renamed the project to `taopq`.

@uilianries Could you have a look if I missed something wrt Conan? Thanks!

**Comment by uilianries** [comment]
Sure! I'll take a look

**Comment by uilianries** [comment]
All looks good!

I'm going to provide a new PR with CI support for Conan.

**Comment by emmenlau** [comment]
Whoa cool! Thanks a lot for your consideration!!!
